### PR TITLE
Doc: L3LS Example and Mkdoc Updates

### PR DESCRIPTION
**Pull request by JulioPDX**
## Change Summary

Refactoring example to follow concepts used in l2ls example. Diagrams, content tabs, etc.

[Read the docs site](https://ansible-avd-juliopdx.readthedocs.io/en/l3ls-doc-refactor/examples/single-dc-l3ls/index.html)

## Proposed changes

Updating the following:

- Layout
- Diagrams
- code blocks inherit from group var files
- mkdocs bump from 8.3.9 to 8.5.1
- Fixed smart symbols rendering fractions
- enabled emoji support

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by JulioPDX** [comment]
@mthiel117 Please give this a review when you have some cycles! Thank you

File: ansible_collections/arista/avd/examples/single-dc-l3ls/README.md [hunk 1]
```diff
@@ -256,86 +228,91 @@ all:
...
-type: l2leaf
-```
+    ```yaml
+    --8<--
+    examples/single-dc-l3ls/group_vars/DC1_L2_LEAVES.yml
+    --8<--
+    ```
 
-For example, this means that all switches that are children of the DC1_SPINES group defined in the inventory will be of the type `spine`.
+For example, all switches that are children of the DC1_SPINES group defined in the inventory will be of the type `spine`.
```
Comment (gmuloc) [inline ansible_collections/arista/avd/examples/single-dc-l3ls/README.md]:
```suggestion
For example, all switches that are children of the DC1_SPINES group defined in the inventory will be of type `spine`.
```

File: ansible_collections/arista/avd/examples/single-dc-l3ls/README.md [hunk 2]
```diff
@@ -256,86 +228,91 @@ all:
...
+ansible_httpapi_validate_certs: false # (6)!
 ```
 
-- The Ansible host must use eAPI via HTTPS.
-- Network OS which in this case is Arista EOS.
-- The username/password combo.
-- How to escalate privileges to get write access.
-- Use SSL.
-- Do not validate SSL certificates.
+1. The Ansible host must use eAPI via HTTPS
```
Comment (gmuloc) [inline ansible_collections/arista/avd/examples/single-dc-l3ls/README.md]: technically does not say use HTTPS - only HTTPAPI

File: ansible_collections/arista/avd/examples/single-dc-l3ls/playbook.yml [hunk 3]
```diff
@@ -1,14 +1,14 @@
 ---
-- name: Run AVD
+- name: Run AVD # (1)!
```
Comment (gmuloc) [inline ansible_collections/arista/avd/examples/single-dc-l3ls/playbook.yml:3]: I understand why the comments are here but out of context it looks weird (though when included in the README.md it does make a lot of sense) - not sure if we could add some additional comment in this file explaining why the comments are here.

**Review by gmuloc — APPROVED** [review]
Looks good! a couple of comments but cosmetic changes so still approving.

File: ansible_collections/arista/avd/examples/single-dc-l3ls/README.md (hunk 2 above)
Comment (JulioPDX) [reply ansible_collections/arista/avd/examples/single-dc-l3ls/README.md]: I'll update this to state "The Ansible host must use eAPI", thank you

File: ansible_collections/arista/avd/examples/single-dc-l3ls/playbook.yml (hunk 3 above)
Comment (JulioPDX) [reply ansible_collections/arista/avd/examples/single-dc-l3ls/playbook.yml:3]: I can add a comment at the root of the playbook file.

File: ansible_collections/arista/avd/examples/single-dc-l3ls/README.md [hunk 4]
```diff
@@ -1,19 +1,16 @@
-# AVD Example for a Single Data Center Using Layer 3 Leaf Spine (L3LS)
+# AVD example for a single data center using layer 3 leaf spine (L3LS)
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/examples/single-dc-l3ls/README.md]: Would this be more correct?
```suggestion
# AVD example for a single data center using Layer 3 Leaf Spine (L3LS)
```

**Review by ClausHolbechArista — APPROVED** [review]
LGTM - a minor comment on the title since I think spelling out an abbreviation should use Title Case.

File: ansible_collections/arista/avd/examples/single-dc-l3ls/README.md (hunk 4 above)
Comment (JulioPDX) [reply ansible_collections/arista/avd/examples/single-dc-l3ls/README.md]: @ClausHolbechArista I think we can even just revert back to L3LS, I'll get this updated

**Pull request merged**